src/components/BoxApproval/BoxApproval.jsx

```diff
@@ -141,16 +141,22 @@ const BoxApproval = () => {
         </Tabs>
         <div className={styles.legend}>
```

Comment:
This legend looks a bit wonky on mobile view - can we style this as display flex and remove all the translate, absolute position, and setting left/top to 50%?

Reply:
yes mommy

Reply:
ok daughter